napalm_ios/ios.py

```diff
@@ -27,6 +27,7 @@
 from napalm_base.exceptions import ReplaceConfigException, MergeConfigException
 from napalm_base.utils import py23_compat
 import napalm_base.constants as C
+import napalm_base.helpers
```

Comment:
I took some direction on the helper implementation from the nxos driver. However there's still some inconsistency even within the nxos driver on whether `convert` is called vs calling `helpers.mac()` directly. 

https://github.com/napalm-automation/napalm-nxos/blob/develop/napalm_nxos/nxos.py#L393-L394

I chose to go the `convert` route as it appears to have some additional error-handling built in. 

https://github.com/napalm-automation/napalm-base/blob/develop/napalm_base/helpers.py#L180-L181